### ISSUE TITLE
Windows 7 Support

### ISSUE THREAD

**Issue by jjolley** [PlayFab/PlayFabPartyUnity#35]
Does this library have Windows 7 Support? We are getting our users reporting the following error:

DllNotFoundException: PartyWin32
  at (wrapper managed-to-native) PartyCSharpSDK.Interop.PFPInterop.PartyInitialize(byte[],PartyCSharpSDK.Interop.PARTY_HANDLE&)
  at PartyCSharpSDK.SDK.PartyInitialize (System.String titleId, PartyCSharpSDK.PARTY_HANDLE& handle) [0x00006] in <d11ff22ed5bc4797b748587f0e552cae>:0 
  at PlayFab.Party.PlayFabMultiplayerManager.InitializeImpl () [0x000be] in <d11ff22ed5bc4797b748587f0e552cae>:0 
  at PlayFab.Party.PlayFabMultiplayerManager._Initialize () [0x0000a] in <d11ff22ed5bc4797b748587f0e552cae>:0 
  at PlayFab.Party.PlayFabMultiplayerManager.Start () [0x00000] in <d11ff22ed5bc4797b748587f0e552cae>:0

**Comment by nassosterz-ms** [comment]
Nope, this library supports Windows 10. We understand that the naming might cause confusion.

Is this issue on Win7?

**Comment by jjolley** [comment]
Yes it is.

**Comment by nassosterz-ms** [comment]
What you could do would be to obtain the Windows 7 binaries of PlayFab Party and replace the Win10 binaries with those.

Please note this: https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/release-notes#:~:text=Support%20for%20the,PlayFab%20Party%20library.
